helpers/leagueHelper.mjs

```diff
@@ -5,14 +5,15 @@ import { mapTransactionPlayers } from "./transactionHelper.mjs";
  * Helper function to map data to a "team"
  */
 export function mapTeams(ts) {
-  const count = ts.count;
-  const teams = [];
+  const teams = Object.values(ts);
 
-  for (let i = 0; i < count; i++) {
-    teams.push(mapTeam(ts[i].team[0]));
-  }
+  return teams.reduce((result, t) => {
+    if (t.team) {
+      result.push(t.team[0]);
+    }
 
-  return teams;
+    return result;
+  }, []);
 }
```

Comment:
This doesn't appear to be working as intended... data coming back is not mapped properly...
```
"teams": [
    [
      {
        "team_key": "388.l.4023.t.1"
      },
      {
        "team_id": "1"
      },
      {
        "name": "chicksDigtheLongball"
      },
      {
        "is_owned_by_current_login": 1
      },
      {
        "url": "https://baseball.fantasysports.yahoo.com/b1/4023/1"
      },
      {
        "team_logos": [
          {
            "team_logo": {
              "size": "large",
              "url": "https://ct.yimg.com/cy/1551/24925833379_5aa9877084_192sq.jpg?ct=fantasy"
            }
          }
        ]
      },
      [],
      {
        "waiver_priority": 12
      },
      {
        "faab_balance": "1"
      },
      {
        "number_of_moves": "51"
      },
      {
        "number_of_trades": "2"
      },
      {
        "roster_adds": {
          "coverage_type": "week",
          "coverage_value": "25",
          "value": "0"
        }
      },
      {
        "clinched_playoffs": 1
      },
      {
        "league_scoring_type": "head"
      },
      [],
      [],
      {
        "has_draft_grade": 0
      },
      {
        "auction_budget_total": "260"
      },
      {
        "auction_budget_spent": 260
      },
      {
        "managers": [
          {
            "manager": {
              "manager_id": "1",
              "nickname": "dewberry",
              "guid": "RYWP7M53IC626MGOX36ZWCM4FA",
              "is_commissioner": "1",
              "is_current_login": "1",
              "email": "dewey@whatadewitt.com",
              "image_url": "https://s.yimg.com/ag/images/4596/39177950812_24b77b_64sq.jpg"
            }
          }
        ]
      }
    ],```